### PR TITLE
Added truncate in breadcrumbs on mobile view

### LINKED ISSUE #3136: Breadcrumbs extend beyond screen in mobile view
https://care.coronasafe.in/facility/05fcf441-48b2-4d90-9389-f9ab9af107d5
https://www.loom.com/share/63661f938f61402ba2fc847c71b57615

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #3136 
http://localhost:4000/facility/c92cc4df-0fea-4d37-9c24-1c9f07bfac23/patient/cee24a4f-7d4c-40ee-8667-cdd6bca99003/consultation/a15c4440-0135-450d-8c48-ef4b99c0e0e6

image

### CODE REVIEW

**Review by cp-Coder — APPROVED** [review]

File: src/Components/Common/Breadcrumbs.tsx [hunk 1]
```diff
@@ -140,7 +140,12 @@ export default function Breadcrumbs(props: any) {
                         </div>
                       ) : (
                         <div>
-                          <span>{crumb.name}</span>
+                          <span className="hidden md:block">{crumb.name}</span>
+                          <span className="block md:hidden">
+                            {crumb.name.length > 13
+                              ? crumb.name.slice(0, 13) + "..."
+                              : crumb.name}
+                          </span>
```
Comment (developedBySJ) [inline src/Components/Common/Breadcrumbs.tsx]: @Pranshu1902 Use  [`truncate`](https://tailwindcss.com/docs/text-overflow#truncate) class instead

**Review by developedBySJ — CHANGES_REQUESTED** [review]

File: src/Components/Common/Breadcrumbs.tsx (hunk 1 above)
Comment (Pranshu1902) [reply src/Components/Common/Breadcrumbs.tsx]: Done

**Comment by Pranshu1902** [comment]
Updated view:
image

**Comment by nihal467** [comment]
@Pranshu1902 
image

unable to see the breadcrumbs in mobile view ( galaxy fold )

**Comment by Pranshu1902** [comment]
> @Pranshu1902 image
> 
> unable to see the breadcrumbs in mobile view ( galaxy fold )

I didn't get it, the breadcrumbs are visible in the screenshot, please elaborate what's the error

**Comment by nihal467** [comment]
@Pranshu1902 
image

the breadcrumbs makes the page lengthy, can we do anything regarding that !

**Comment by Pranshu1902** [comment]
@nihal467 
Updated view:
image

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**